javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $container = document.querySelector('.container.on');
+var $lightbulb = document.querySelector('.lightbulb.on');
+
+function clickLightbulb(event) {
+  if ($lightbulb.className === 'lightbulb on') {
```

Comment:
Instead of updating based on a value on the DOM, I would recommend a variable here in the JS to determine if the light is on or off. A good way to accomplish that is a variable named something like `isOn` that holds a boolean value